docs/general/support-contact.md

```diff
@@ -4,30 +4,100 @@ hide:
   - toc
 ---
 
-### Support Contact
+### Our Support Team
 
-Our team is available for online support from Monday to Friday between 8.00 and 18.00 CET.
-In case of technical issues, you can reach out to our support team via email on <support@121.global>. Please, always inform or add your Account Manager in copy of your requests, so they can follow up with you. Our response time is 1-2 hour on average.
+Our team consists of Cash IM practitioners with extensive experience in Cash and Voucher Assistance (CVA) programs, as well as experts in software, data and digital solutions. Most of our team members have been deployed to support global humanitarian aid operations, working within the Red Cross Red Crescent Movement and other international NGOs. We bring expertise in preparedness, localization, disaster response, and emergency crisis management.
 
-!!! Warning "Urgent technical issues"
-    For urgent technical issues, you can contact us directly via WhatsApp or Teams, or other agreed means of instant communication.
+### How to Contact Support
+
+For general support inquiries, your first point of contact is always your Account Manager. If further technical assistance is required, the issue will be smoothly escalated to the appropriate team.
+
+<!-- markdownlint-disable -->
+**Submit Requests to**
+<a href="mailto:support@121.global" style="text-decoration: none;">
+  <button style="background-color: #003366; color: #fff; padding: 12px 24px; border: none; border-radius: 25px; font-size: 16px; font-family: Arial, sans-serif; cursor: pointer; box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); transition: background-color 0.3s ease;">
+    📧 support@121.global
+  </button>
+</a>
```

Comment:
`<button>`-elements cannot be inside `<a>`/link-elements. (Both handle 'clicks', so they cannot be mixed. 

But it seems that in this case "a button-like shaped link to our e-mail address" is wanted. 

So we could create a specific set of CSS-rules in the `extra.css`-file. (Including some specific/styled icon)
So the only thing necessary would be:

```
<a href="mailto:support@121.global" class"big-attention-grabbing-button">support@121.global</a>
```

It would still require the "markdownlint-ignore"-line though... But we could limit it to "this specific rule only", so we don't miss any other "real errors", besides this.

Reply:
Ok, scratch my previous suggestions... :)

There's a 'component' for it out-of-the-box: https://squidfunk.github.io/mkdocs-material/reference/buttons/#usage

And icons: https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/